src/test/java/ikysil/training/ws/api/v1/OrdersResourceTest.java

```diff
@@ -24,27 +30,170 @@
 @RunWith(SpringRunner.class)
 public class OrdersResourceTest {
 
+    @Autowired
+    OrderDao dao;
+
     public static final String BASE_URL = "/ws/api/v1/orders/";
 
     @LocalServerPort
     private int port;
 
+    private OrderDto order1 = new OrderDto("ikl11444", "twenty one", "none", "3129", "firstuser@gmail.com");
+    private OrderDto order2 = new OrderDto("qwe32577", "twenty two", "none", "4024", "seconduser@gmail.com");
+    private OrderDto newOrder = new OrderDto("qwe32577","thirty three", "noneone", "2440", "newuser@gmail.com");
+
     @Before
     public void setUp() {
         RestAssured.basePath = BASE_URL;
         RestAssured.port = port;
+        dao.saveOrder(order1);
+        dao.saveOrder(order2);
     }
 
     @Test
-    // TODO rename test
-    public void test() {
-        OrderDto request = new OrderDto();
-
+    public void savingOKTest() {
```

Comment:
ok, you did two `POST` requests, checked response status code. 
But as well would be good to ensure those orders in DB through `dao...`.